src/main/java/com/amihaiemil/docker/Docker.java

```diff
@@ -44,6 +44,12 @@
      */
     boolean ping() throws IOException;
 
+    /**
+     * Version of the Docker API.
+     * @return String version, for instance "v1.30".
+     */
+    String version();
```

Comment:
@amihaiemil what's the reason for adding `version()`? The user already knows the version because it's parametrized in both `LocalDocker` and `RemoteDocker`

Reply:
@amihaiemil this library has no factories

Reply:
@llorllale I know :)) 

I am referring to the client code, which may have a ``Docker`` instance from somewhere outside its scope and may want to see what version is this ``Docker instance. 

You really don't see a purpose? If you think I should remove it, I"ll remove it, you are the CR :D